src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html

```diff
@@ -11,21 +12,43 @@
     [tabIndex]="-1"
   >
     {{ groupHeading }}
-    <sky-chevron *skyThemeIf="'default'"
+    <sky-chevron
```

Comment:
I'm curious about these chevron changes from a design standpoint. 

The change from up-down to right-down was intentionally made when we converted this to modern theme. This seems to be undoing that.

Reply:
Also a note that if we are changing the chevron directions back then I don't think we need the `skyThemeIf` parts at all as we would then just have functionally the same things and should just be able to use the `sky-chevron` for both themes. We only had the `skyThemeIf` stuff because the `sky-chevron` didn't support the deliberate move we made to change the direction to `right` in modern theme.

Reply:
@Blackbaud-TrevorBurch Gotcha on the `sky-chevron`. I switched this back. It just means there wasn't a way to make the selected group's arrow the selected color but not a biggie. 
I still kept the `*skyThemei`f sections b/c the `<div>` needs a different font class when the sub menu is open in modern. If there is a way to use` [SkyThemeClass]` with that dynamic state `: subMenuOpen()` then I could just have the classes change based on the theme instead of the whole `div`.

Reply:
Updated to just have `sky-chevron` for both themes. Switching back to up/down arrow for representing expanded/collapsed for the accordion control.